applications/DEMApplication/custom_constitutive/DEM_global_damping_model.h

```diff
@@ -0,0 +1,50 @@
+//  Kratos Multi-Physics - DEM Application
+//
+//  License:       BSD License
+//                 Kratos default license: kratos/license.txt
+//
+//  Main authors:  Rafael Rangel (rrangel@cimne.upc.edu)
+//
+
+#pragma once
+
+// Includes
+#include "includes/define.h"
+#include "containers/array_1d.h"
+#include "custom_utilities/AuxiliaryFunctions.h"
+
+namespace Kratos
+{
+    class SphericParticle; // Forward declaration of spheric particle
+
+    class KRATOS_API(DEM_APPLICATION) DEMGlobalDampingModel
+    {
+        public:
+            // Pointer definition
+            KRATOS_CLASS_POINTER_DEFINITION(DEMGlobalDampingModel);
+
+            // Constructor / destructor methods
+            DEMGlobalDampingModel() {}
+            virtual ~DEMGlobalDampingModel() {}
+
+            // Clone
+            virtual DEMGlobalDampingModel::Pointer Clone() const;
+            virtual std::unique_ptr<DEMGlobalDampingModel> CloneUnique();
+
+            // Public methods
+            virtual void SetGlobalDampingModelInProperties(Properties::Pointer pProp, bool verbose = true);
+            virtual void AddGlobalDampingForceAndMoment(SphericParticle* p_element, array_1d<double,3>& total_forces, array_1d<double,3>& total_moment) {}
```

Comment:
Maybe you can add the following here:
KRATOS_ERROR << "This function (....) shouldn't be accessed, use derived class instead"<<std::endl;"